TKeazirian.HTTPServer/Response/ResponseBuilder.cs

```diff
@@ -21,16 +21,15 @@ public ResponseBuilder SetHeaders(string headerName, string headerValue)
         return this;
     }
 
-    public ResponseBuilder SetBody(string body)
+    public ResponseBuilder SetBodyBytes(byte[]? bodyBytes)
```

Comment:
I would expect to handle text as text and bytes as bytes. In other words, to extend this class.  Instead it appears that you have replaced one for the other.  How can you support both?

Alternatively, if you must replace one with the other, then why all the renaming to include `Bytes`? It is causing a cascade of updates throughout the entire codebase. Functions are "black boxes", you should feel empowered to change the internals without having to rename.

Reply:
I suggest keeping the distinction between `Body` and `BodyBytes` if you are supporting two things.  Otherwise, just keep using `Body`.

Reply:
Unless there is a convincing argument to the contrary, I also suggest supporting both text (text, html, xml, json) and bytes (images).